internals/plan/plan.go

```diff
@@ -963,6 +925,15 @@ func ParseLayer(order int, label string, data []byte) (*Layer, error) {
 				Message: fmt.Sprintf("cannot use reserved service name %q", name),
 			}
 		}
+		// Deprecated service names
+		if name == "all" || name == "default" || name == "none" {
+			logger.Noticef("WARNING: using %q as service name is deprecated", name)
```

Comment:
As a rule of thumb, error messages are lowercased and use no ending punctuation, while log messages are proper sentences starting with capital case and ending with punctuation.